src/main/resources/application.properties

```diff
@@ -0,0 +1,12 @@
+quarkus.datasource.jdbc.url=jdbc:postgresql://localhost:5432/Customer
+quarkus.datasource.jdbc.driver=org.postgresql.Driver
```

Comment:
Parameterize the values so that we can inject the values for higher environments. Eg: quarkus.datasource.username=${dbusername:postgres}